app/src/userpages/components/StreamPage/List/index.jsx

```diff
@@ -288,13 +294,31 @@ class StreamList extends Component<Props, State> {
                             </thead>
                             <tbody>
                                 {streams.map((stream) => (
-                                    <tr key={stream.id}>
-                                        <th>{stream.name}</th>
-                                        <td title={stream.description}>{stream.description}</td>
-                                        <td>{moment(stream.lastUpdated).fromNow()}</td>
-                                        <td>-</td>
-                                        <td><StatusIcon /></td>
-                                        <td>
+                                    <tr
+                                        key={stream.id}
+                                        className={styles.streamRow}
+                                    >
+                                        <th onClick={() => this.onStreamRowClick(stream.id)}>{stream.name}</th>
+                                        {/* eslint-disable-next-line max-len */}
+                                        {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions */}
```

Comment:
I suggest disabling this rule for the entire block rather than around each instance:
```jsx
<tr>
  {/* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions */}
  <td /> 
  …
  {/* eslint-enable jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions */}
</tr>
```